palantirnet/tester
# Created info.yml and composer.json files for PHP 8 tester

## JIRA Ticket(s)

- See ticket [PHP-1: Create Drupal 9 module info and drush registry](https://palantir.atlassian.net/browse/PHP-1).

## Description

Create a tester.info.yml file with required information, create composer.json with a services file with required Drush integration

## Testing instructions

Check to see if there are any missing lines of code that will be added to composer.json and tester.info.yml files

## Relevant Screenshots/gifs:

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
